paradox-nakadi-consumer-core/src/main/java/de/zalando/paradox/nakadi/consumer/core/http/handlers/EventUtils.java

```diff
@@ -43,7 +45,13 @@ private EventUtils() { }
                     if (!element.isNull()) {
                         String rawEvent = null;
                         try {
-                            rawEvent = jsonMapper.writeValueAsString(element);
+                            if (!element.get("metadata").has("event_type")
```

Comment:
If the element does not have "metadata" field, it will throw NPE.
See: https://fasterxml.github.io/jackson-databind/javadoc/2.0.0/com/fasterxml/jackson/databind/node/ArrayNode.html#get(java.lang.String)